src/schema/users.ts

```diff
@@ -2269,14 +2336,19 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         flags,
       });
 
-      await Promise.all([
-        resubscribeUser(cio, ctx.userId),
-        identifyUserPersonalizedDigest({
-          userId: ctx.userId,
-          cio,
-          subscribed: true,
-        }),
-      ]);
+      const promises = [resubscribeUser(cio, ctx.userId)];
+
+      if (process.env.NODE_ENV !== 'development') {
```

Comment:
I think these checks should be handled within the function itself, not on every instance where we call it.